src/main/scala/io/iohk/ethereum/jsonrpc/JsonMethodsImplicits.scala

```diff
@@ -38,7 +38,7 @@ trait JsonMethodsImplicits {
     }.toEither.left.map(_ => InvalidParams())
 
   private def decode(s: String): Either[JsonRpcError, Array[Byte]] = {
-    if(!s.startsWith("0x")) Left(InvalidParams())
+    if(!s.isEmpty && !s.startsWith("0x")) Left(InvalidParams())
```

Comment:
Shit :disappointed:  ... I will check it myself because I've been asked to change to the current implementation. 

BTW, if other clients didn't follow that spec, does it mean we shouldn't. I understand some apps won't work. IMO we either need to decide to create a PR to update the docs or be strict following the docs, what do you think?